core/internal/features/ocr2/features_ocr2_test.go

```diff
@@ -275,7 +275,8 @@ schemaVersion		= 1
 contractID			= "%s"
 [relayConfig]
 chainID 			= 1337
-`, ocrContractAddress))
+fromBlock = %d
+`, ocrContractAddress, blockBeforeConfig.Number().Int64()))
```

Comment:
Wondering if we should add this to the OCR2VRF integration tests as well, see `core/services/ocr2/plugins/ocr2vrf/internal/ocr2vrf_integration_test.go`.

Reply:
Looks like this removes the need of doing an explicit replay, which is quite useful for the tests.